src/metatensor/models/utils/external_naming.py

```diff
@@ -0,0 +1,71 @@
+from typing import Dict, Union
+
+from metatensor.torch.atomistic import ModelOutput
+
+
+def to_external_name(
```

Comment:
I don't understand why this is required, and I don't love that we are changing the names of stuff behind the user back. Why are we naming things `energy_positions_gradients` instead of using `energy = TensorBlock(...); forces = energy.gradient("positions")`?

Reply:
In my opinion, that's too many assumptions and hardcoding for a library that is supposed to handle arbitrary targets

Reply:
I thought the user has to use the "correct" names in this `options.yaml` and then we should stick with these names.